ChangeLog.md

```diff
@@ -1,6 +1,139 @@
 
 # Change Log
 
+## Version 10.23.0
```

Comment:
Why are some of these changes showing in the diff even though they're in main?

Reply:
Good question

Reply:
i think it got resolved by your last commit, dw about it